File: 2st/Riyeon/홀짝에 따라 다른 값 반환하기.py
def solution(n):
    eo = 0 if n % 2 == 0 else 1
    answer = 0
    for i in range(n+1):
        if i % 2 == eo:
            if eo == 1:
                answer += i
            else:
                answer += i*i
    return answer

File: 4st/Gau/programmer/홀짝에 따라 다른 값 반환하기.java
class Solution {
    public int solution(int n) {
        int a = 0;
        if(n%2 == 1){
            for(int i=1;i<=n;i+=2){
                a += i;
            }
        }
        else{
            for(int i=2;i<=n;i+=2){
                a += i*i;
            }
        }
        return a;
    }
}